docs/code_conventions/README.MD

```diff
@@ -34,6 +34,41 @@ apiVersion = "v1"
 username = demisto.params().get("url")
 ```
 
+## Arguments
+When naming an argument, the name should be all lowercase with spaces replaced by underscores. See the example below:
+
+### Do this:
+```
+email_header
+```
+ ### Not this:
+ ```python
+Email Header
+```
+
+Additionally, arguments are to be passed through the ```_command``` function and mustn't reside in the primary function. Observe the following:
+
+**Do This**
+
+```python
+def sample_function(arg1):
+    response = arg1 + 'is the argument'
+    return response
+
+def sample_function_command():
+    arg1 = demisto.args().get('arg1')
+    response = sample_function(arg1)
+    demisto.results('This is the response - {}'.format(response))
+```
+
+**Not This**
+```python
+def sample_function():
+    arg1 = demisto.args().get('arg1')
+    response = arg1 + 'is the argument'
+    demisto.results('This is the response - {}'.format(response))
+```
+
 ## Command Functions
 Command functions should be split into two functions. We will look at "sample_function" as an example below:
 
```

Comment:
can't command on full files:
line 78: add a return value
line 106 (helper function section): remove the list parameter
line 158 (`Logging` section): double "ensure" word
`Catch Me If You Can` section: remove the LOG functions in the except part, they are part of the `return_error` (new in master).
`Common Server Functions` section: before writing, not after ;)
line 264: `url = ...`